server/svix-server/src/v1/endpoints/endpoint/mod.rs

```diff
@@ -96,37 +96,28 @@ fn validate_channels_endpoint_unrequired_nullable(
     }
 }
 
-pub fn validate_url(val: &str) -> std::result::Result<(), ValidationError> {
-    match Url::parse(val) {
-        Ok(url) => {
-            let scheme = url.scheme();
-            if scheme == "https" || scheme == "http" {
-                Ok(())
-            } else {
-                Err(validation_error(
-                    Some("url"),
-                    Some("Endpoint URL schemes must be http or https"),
-                ))
-            }
-        }
+pub fn validate_url(url: &Url) -> std::result::Result<(), ValidationError> {
+    tracing::warn!("validating url = {:?}", url.to_string());
```

Comment:
Why is there a `warn` log here?

Reply:
Because I forgot to delete after debugging locally. Deleting . . .